client/src/lib.rs

```diff
@@ -175,18 +182,31 @@ impl Default for TurnkeyClientBuilder {
 }
 
 /// Base client. To create a new client, see `TurnkeyClient::builder`.
-#[derive(Debug)]
 pub struct TurnkeyClient {
     http: reqwest::Client,
     base_url: String,
-    api_key: TurnkeyP256ApiKey,
+    api_key: Box<dyn ApiKeyStamper>,
```

Comment:
This could be e.g. a generic parameter instead of a box. or `impl ApiKeyStamper`